nursery/get-system-information-via-wmi-in-dotnet.yml

```diff
@@ -0,0 +1,20 @@
+rule:
+  meta:
+    name: get system information via WMI in .NET
```

Comment:
WMI can do other things than just get system information. Adjust the name here?

Reply:
`access WMI data in .NET`?

Reply:
sounds good